agent/src/agent/cli/streamsets.py

```diff
@@ -34,11 +34,12 @@ def list_():
 @click.option('--username', type=click.STRING, default='admin')
 @click.option('--password', type=click.STRING, default='admin')
 @click.option('--agent-ext-url', type=click.STRING, default='http://anodot-agent')
-def add(url, username, password, agent_ext_url):
+@click.option('--preferred-type', type=click.STRING, default=None)
+def add(url, username, password, agent_ext_url, preferred_type):
```

Comment:
please also add prompt option here, for configuring streamsetsts we use cli mostly